### ISSUE TITLE
feat: create mnemonics

### ISSUE THREAD

**Issue by aesedepece** [witnet/sheikah#220]
The renderer process needs a simple function with this contract:
```ts
type NewMnemonicsResponse = {
  mnemonics: string
}

function newMnemonics(): Promise<NewMnemonicsResponse>
```
This function makes the API client issue a request with method "`newMnemonics`" and no params (`[]`).

The `newMnemonics` handler:
+ uses @kronolynx's code for mnemonics generation (see `app/main/crypto/mnemonic.ts:generate()`),
+ creates an "unconsolidated wallet entry" containing the mnemonic in main process' app state,
+ returns the unconsolidated wallet entry.

**Comment by aesedepece** [comment]
The shape of the unconsolidated wallets looks like this:
```ts
type UnconsolidatedWallet = Partial<{
  id: string
  caption?: string
  mnemonics: string
}>
```

**Comment by aesedepece** [comment]
Closed by #240
